doc/lsst.ap.verify/datasets-creation.rst

```diff
@@ -28,12 +28,26 @@ Organizing the Data
 
 * The :file:`raw` and :file:`calib` directories contain science and calibration data, respectively.
   The directories may have any internal structure.
-* The :file:`templates` directory contains an :ref:`LSST Butler repository<butler>` containing processed images useable as templates.
+* The :file:`templates` directory contains an :ref:`LSST Butler repository<butler>` containing processed images usable as templates.
   Template files must be ``TemplateCoadd`` files produced by a compatible version of the LSST science pipeline.
 * The :file:`refcats` directory contains one or more tar files, each containing one astrometric or photometric reference catalog in HTM shard format.
 
 The templates and reference catalogs need not be all-sky, but should cover the combined footprint of all the raw images.
 
+.. _ap-verify-datasets-creation-docs:
+
+Documenting Datasets
+--------------------
+
+Datasets provide package-level documentation in their :file:`doc` directory.
+An example is provided in the `dataset template repository`_.
+
+The dataset's package-level documentation should include:
+
+* the source of the data (e.g., a particular survey with specific cuts applied)
+* whether or not optional files such as templates are provided
```

Comment:
"Template" may be a bit overloaded here. Clarify that you mean coadds and/or visits (i.e., template images) for difference imaging when you say templates, please.

Reply:
"Image differencing templates"?